tests/containers/covid19/Covid19Container-test.jsx

```diff
@@ -10,8 +10,8 @@ import * as apis from 'apis/disaster';
 import * as actions from 'redux/actions/covid19/covid19Actions';
 import { useQueryParams } from 'helpers/queryParams';
 import Covid19Container from 'containers/covid19/Covid19Container';
-import { mockDefCodes } from '../../mockData/helpers/disasterHelper';
-import { defaultState } from '../../testResources/defaultReduxFilters';
+import { mockDefCodes, mockDefcParams } from '../../mockData/helpers/disasterHelper';
+// import { defaultState, covid19DefaultState } from '../../testResources/defaultReduxFilters';
```

Comment:
Can this line be removed instead of commented?

Reply:
yes, sorry, left over from trying to figure a problem out